test/task_test.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+require_relative 'modbus_helpers'
+
+using_task_library 'motors_weg_cvw300'
+
+describe OroGen.motors_weg_cvw300.Task do
+    run_live
+
+    attr_reader :task
+    attr_reader :reader
+
+    include ModbusHelpers
+
+    before do
+        @task, @reader, @writer = iodrivers_base_prepare(
```

Comment:
It looks a bit odd to me that you use `@reader` and `@writer` in the module but only set them here..

Reply:
Yeah, I agree. It's pretty implicit. Changed it.